addons/analytic/static/src/components/analytic_distribution/analytic_distribution.js

```diff
@@ -72,6 +73,54 @@ export class AnalyticDistribution extends Component {
         this.allPlans = [];
         this.lastAccount = this.props.account_field && this.props.record.data[this.props.account_field] || false;
         this.lastProduct = this.props.product_field && this.props.record.data[this.props.product_field] || false;
+
+        this.activeTag = null;
```

Comment:
I'm not sure I understand the logic, it seems a bit dirty.
Moreover, we set the focus on the first element but when we press tab, it goes directly to after the last account we selected (so jumping from first to new).
It would be nice to be able to tab quickly to edit the amounts

Reply:
I need to keep track of the tag that was edited (could have been an existing tag)
The tag id will have changed if the popup closes and opens again - therefore I record the `activeTag` so I know which one to replace.
The tabs are fixed now (since the balance is filled to the first 0 value unedited tag)

Reply:
removed (editedTag is passed through the functions in the stack)